content/livesync/index.textile

```diff
@@ -1,6 +1,57 @@
 
+This diagram illustrates how the "Ably Realtime Data Models SDK":https://https://github.com/ably-labs/models, "ADBC":https://github.com/ably-labs/adbc and "Outbox Table":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html work together:
 
+<a href="/images/diagrams/datasync-e2e-solution.png" target="_blank">
+  <img src="/images/diagrams/datasync-e2e-solution.png" style="width: 100%" alt="Basic authentication process diagram">
+</a>
+
+- User interaction := with the application's UI involves entering data stored in the local state.
+- Optimistic update := means the update immediately reflects the user's changes in the UI.
+- Local State := changes made by the user are first written to the application's local state, which then sends these changes to the server.
+- Server API := sends a write request to the server, which processes this request through an API endpoint. The server then attempts to write the changes in a transactional form to the database.
+- The database's := new state is saved in database tables.
+- A confirmed State := is then returned to the application from the server, signaling that the changes have been successfully persisted.
+- "Ably Realtime Data Models SDK":https://https://github.com/ably-labs/models := includes tools and libraries to handle the data models and synchronize with the server.
+- Subscription to change := allows the app to receive updates about confirmed state in realtime
+
+h2. LiveSync’s Key Capabilities
+
+* *Realtime database synchronisation* integrates clients with relational databases in realtime. The primary reason for using LiveSync is it directly impacts the user experience by providing immediate data updates between multiple clients.
+
+* *Data integrity and delivery guarantees* guarantees data integrity and the exact-once delivery of synchronised data changes and user actions.
+
+* *Stateful interactive applications* to maintain consistency across multiple clients effortlessly.
+
+* *Optimistic updates at  API Level* makes updates at the API level, LiveSync offers users an immediate view of changes, eliminating the wait for backend confirmation.
+
+* *Scalability and reliability* means LiveSync manages large user spikes and interactive actions reliably, supported by a robust uptime service-level agreement (SLA).
+
+* *LiveSync alerts subscribers* to new data, eliminating server polling.
+
+* *Integration with all databases* designed to work with any type of database you prefer.
```

Comment:
Currently this is only Postgres, and specifically we've designed around transactional stores at this point